pkg/terra/config.go

```diff
@@ -0,0 +1,143 @@
+package terra
+
+import (
+	"sync"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/smartcontractkit/chainlink-terra/pkg/terra/db"
+)
+
+var DefaultConfigSet = ConfigSet{
+	// ~8s per block, so ~80s until we give up on the tx getting confirmed
+	// Anecdotally it appears anything more than 4 blocks would be an extremely long wait.
+	BlocksUntilTxTimeout:  10,
+	ConfirmMaxPolls:       100,
+	ConfirmPollPeriod:     time.Second,
+	FallbackGasPriceULuna: sdk.MustNewDecFromStr("0.01"),
```

Comment:
See comment here https://github.com/smartcontractkit/chainlink-terra/pull/32/files#r787938891 on whether we want to keep this. Also we should increase it to 0.015